### PR TITLE
Code snippets for Rooms REST API - Participants and PublishedTrack

### PR DESCRIPTION
**Pull request by kedartoraskar**

### CODE REVIEW

**Comment by kedartoraskar** [comment]
@innerverse @atbaker please review.

File: video/rooms/participants/kick-remove-participant/remove-participant.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import com.twilio.Twilio;
+import com.twilio.rest.video.v1.room.Participant;
+
+public class ParticipantsExample2 {
+	public static final String API_KEY_SID = "SKXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+	public static final String API_KEY_SECRET = "your_api_key_secret";
+
+	public static void main(String args[]) {
```
Comment (innerverse) [inline video/rooms/participants/kick-remove-participant/remove-participant.java]: Can we use "4 spaces" in all java files?  This appears to be 8 spaces..

File: video/rooms/participants/kick-remove-participant/remove-participant.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+// https://www.twilio.com/docs/libraries/node
+var apiKeySid = 'SKXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
+var apiKeySecret = 'your_api_key_secret';
+var accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
+var Twilio = require('twilio');
+
+var client = new Twilio(apiKeySid, apiKeySecret, {accountSid: accountSid});
+
+client.video.rooms('RMXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX').participants.get('Alice').fetch()
+  .then((participant) => {
```
Comment (innerverse) [inline video/rooms/participants/kick-remove-participant/remove-participant.js]: Unnecessary parens -- `.then(participant => `

File: video/rooms/participants/published-track/retrieve-track-published-by-participant/retrieve-track-published-by-participant.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import com.twilio.rest.video.v1.room.participant.PublishedTrack;
+
+public class PublishedTrackExample1 {
+	public static final String API_KEY_SID = "SKXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+	public static final String API_KEY_SECRET = "your_api_key_secret";
+
+	public static void main(String args[]) {
+		Twilio.init(API_KEY_SID, API_KEY_SECRET);
+
+		PublishedTrack publishedTrack = PublishedTrack.fetcher(
```
Comment (innerverse) [inline video/rooms/participants/published-track/retrieve-track-published-by-participant/retrieve-track-published-by-participant.java]: Try this formatting:
```
PublishedTrack publishedTrack = PublishedTrack
        .fetcher("DailyStandup", "Alice", "Camera")
        .fetch();
```

File: video/rooms/participants/published-track/retrieve-track-published-by-participant/retrieve-track-published-by-participant.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+// https://www.twilio.com/docs/libraries/node
+var apiKeySid = 'SKXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
+var apiKeySecret = 'your_api_key_secret';
+var accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
+var Twilio = require('twilio');
+
+var client = new Twilio(apiKeySid, apiKeySecret, {accountSid: accountSid});
+
+client.video.rooms('DailyStandup').participants.get('Alice').publishedTracks.get('Camera').fetch()
+  .then((publishedTrack) => {
```
Comment (innerverse) [inline video/rooms/participants/published-track/retrieve-track-published-by-participant/retrieve-track-published-by-participant.js]: `.then(publishedTrack =>`

File: video/rooms/participants/published-track/retrieve-track-published-by-participant/retrieve-track-published-by-participant.py [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# Download the Python helper library from twilio.com/docs/python/install
+from twilio.rest import Client
+
+# Your Account Sid and Auth Token from twilio.com/console
+api_key_sid = "SKXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+api_key_secret = "your_api_key_secret"
+client = Client(api_key_sid, api_key_secret)
+
+publishedtrack = client.video.rooms('DailyStandup').participants.get(
```
Comment (innerverse) [inline video/rooms/participants/published-track/retrieve-track-published-by-participant/retrieve-track-published-by-participant.py]:
```publishedtrack = client.video.rooms('DailyStandup').participants.get('Alice')
  .published_tracks.get('Camera')
```

File: video/rooms/participants/published-track/retrieve-tracks-published-by-participant/retrieve-tracks-published-by-participant.java [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+import com.twilio.rest.video.v1.room.participant.PublishedTrack;
+
+public class PublishedTrackExample2 {
+	public static final String API_KEY_SID = "SKXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+	public static final String API_KEY_SECRET = "your_api_key_secret";
+
+	public static void main(String args[]) {
+		Twilio.init(API_KEY_SID, API_KEY_SECRET);
+
+		ResourceSet<PublishedTrack> publishedTracks = PublishedTrack.reader(
```
Comment (innerverse) [inline video/rooms/participants/published-track/retrieve-tracks-published-by-participant/retrieve-tracks-published-by-participant.java]: For java, 4 spaces and prefer breaks before `.`

File: video/rooms/participants/published-track/retrieve-tracks-published-by-participant/retrieve-tracks-published-by-participant.js [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+// https://www.twilio.com/docs/libraries/node
+var apiKeySid = 'SKXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
+var apiKeySecret = 'your_api_key_secret';
+var accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
+var Twilio = require('twilio');
+
+var client = new Twilio(apiKeySid, apiKeySecret, {accountSid: accountSid});
+
+client.video.rooms('DailyStandup').participants.get('PAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
+.publishedTracks.list().then((publishedTracks) => {publishedTracks.forEach(publishedTrack => console.log(publishedTrack.sid))});
```
Comment (innerverse) [inline video/rooms/participants/published-track/retrieve-tracks-published-by-participant/retrieve-tracks-published-by-participant.js]: * add 2 spaces before publishedTracks
* remove parens around published tracks here: `.then(publishedTracks =>`
* space or line break after `{`

File: video/rooms/participants/published-track/retrieve-tracks-published-by-participant/retrieve-tracks-published-by-participant.py [hunk 8]
```diff
@@ -0,0 +1,13 @@
+# Download the Python helper library from twilio.com/docs/python/install
+from twilio.rest import Client
+
+# Your Account Sid and Auth Token from twilio.com/console
+api_key_sid = "SKXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+api_key_secret = "your_api_key_secret"
+client = Client(api_key_sid, api_key_secret)
+
+publishedtrack = client.video.rooms('DailyStandup').participants.get(
```
Comment (innerverse) [inline video/rooms/participants/published-track/retrieve-tracks-published-by-participant/retrieve-tracks-published-by-participant.py]:
```
  .get('PAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
  .published_tracks.list()
```

Verify your spacing!  Python should be 4 spaces indent based on Google style code.  This looks like 8.

File: video/rooms/participants/retrieve-participant-list-connected/retrieve-participants-connected.java [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+public class ParticipantsExample3 {
+
+	public static final String API_KEY_SID = "SKXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+	public static final String API_KEY_SECRET = "your_api_key_secret";
+
+	public static void main(String args[]) {
+		Twilio.init(API_KEY_SID, API_KEY_SECRET);
+
+		ResourceSet<Participant> participants = Participant.reader("DailyStandup")
+				.setStatus(Participant.Status.CONNECTED)
```
Comment (innerverse) [inline video/rooms/participants/retrieve-participant-list-connected/retrieve-participants-connected.java]: This looks good but use 4 spaces in java, nice line breaks on `.` here.

File: video/rooms/participants/retrieve-participant-list-connected/retrieve-participants-connected.js [hunk 10]
```diff
@@ -0,0 +1,14 @@
...
+// https://www.twilio.com/docs/libraries/node
+var apiKeySid = 'SKXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
+var apiKeySecret = 'your_api_key_secret';
+var accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
+var Twilio = require('twilio');
+
+var client = new Twilio(apiKeySid, apiKeySecret, {accountSid: accountSid});
+
+client.video.rooms('RMXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX').participants.get('Alice').fetch()
+  .then((participant) => {
```
Comment (innerverse) [inline video/rooms/participants/retrieve-participant-list-connected/retrieve-participants-connected.js]: Remove parents around `participant`

File: video/rooms/participants/retrieve-participant-list-disconnected/retrieve-participants-disconnected.js [hunk 11]
```diff
@@ -0,0 +1,14 @@
...
+// https://www.twilio.com/docs/libraries/node
+var apiKeySid = 'SKXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
+var apiKeySecret = 'your_api_key_secret';
+var accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
+var Twilio = require('twilio');
+
+var client = new Twilio(apiKeySid, apiKeySecret, {accountSid: accountSid});
+
+client.video.rooms('RMXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX').participants.get('Alice').fetch()
+  .then((participant) => {
```
Comment (innerverse) [inline video/rooms/participants/retrieve-participant-list-disconnected/retrieve-participants-disconnected.js]: Remove parents on `(participant)`

File: video/rooms/participants/kick-remove-participant/remove-participant.java (hunk 1 above)
Comment (kedartoraskar) [reply video/rooms/participants/kick-remove-participant/remove-participant.java]: The source files look good in Eclipse or even Sublime. Not sure why it goes to 8 spaces here on GH. Any suggestions ? @innerverse

File: video/rooms/participants/kick-remove-participant/remove-participant.js (hunk 2 above)
Comment (kedartoraskar) [reply video/rooms/participants/kick-remove-participant/remove-participant.js]: Good catch

**Pull request merged**